.github/workflows/learn-github-actions.yml

```diff
@@ -0,0 +1,13 @@
+name: learn-github-actions
+run-name: ${{ github.actor }} is learning GitHub Actions
+on: [push]
```

Comment:
- trigger for this workflow
- push event
  -  triggered every time someone pushes a change
  - merges a pull request
  - triggered by a push to every branch
    - for examples of syntax that runs only on pushes to specific branches, paths, or tags, see "[Workflow syntax for GitHub Actions](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore)."